src/attestation_types/quote.rs

```diff
@@ -155,8 +155,37 @@ impl Default for QuoteHeader {
     }
 }
 
+/// Wrapper struct for the u32 indicating the signature data length
+/// (described in A.4).
+#[derive(Debug, Clone, Copy)]
+#[repr(C)]
+pub struct SigDataLen(u32);
+
+impl SigDataLen {
+    /// Creates a SigDataLen from a u32.
+    pub fn from_u32(val: u32) -> Self {
```

Comment:
Why not `impl From<u32> for SigDataLen {}` ??

Reply:
You had a good point here - that would have been the more idiomatic way to do it. I think I will update it.